src/onegov/org/models/message.py

```diff
@@ -97,7 +97,7 @@ def create(  # type:ignore[override]
         cls,
         ticket: Ticket,
         request: 'OrgRequest',
-        text: str,
+        text: str | None,
```

Comment:
As far as I could tell, there are places that rely on `TicketNote` never being `None`, so you shouldn't change this and either assert that a variable doesn't contain `None` or pass an empty string rather than `None` in places where that's a problem e.g. change from `text=foo` to `text=foo or ''`.

Currently we get the worst of both worlds, because the override in the `TYPE_CHECKING` block above claims that text is `Column[str]` rather than `Column[str | None]`, so we get neither an error for `None` when passing it in or reading it back out.